wezterm-gui/src/main.rs

```diff
@@ -139,6 +139,7 @@ async fn async_run_ssh(opts: SshCommand) -> anyhow::Result<()> {
         always_new_process: true,
         class: opts.class,
         cwd: None,
+        cmd: false,
```

Comment:
maybe replace with `.. Default::default()` construction, so we can avoid referencing the dummy arg?